src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/RethrowToPreserveStackDetailsTests.cs

```diff
@@ -356,6 +356,44 @@ End Class
 ", TestValidationMode.AllowCompileErrors);
         }
 
+        [Fact]
+        [WorkItem(2167, "https://github.com/dotnet/roslyn-analyzers/issues/2167")]
+        public void CA2200_NoDiagnosticsForCatchWithOutArgument()
+        {
+            VerifyCSharp(@"
+using System;
+
+class Program
+{
+    void CatchAndRethrow(Exception exception)
+    {
+        try
+        {            
+        }
+        catch
```

Comment:
❔ Do we already have a test for `catch (Exception)`? If not, it might be good to add one at the same time.

Reply:
Added